### PR TITLE
#20 Add elasticsearch and kibana extension.

### PR DESCRIPTION
**Pull request by hkirsman**
Add Elasticsearch and Kibana extension.

## Testing

1. Require this branch:

`lando composer require wunderio/lando-drupal:dev-feature/20-add-elasticsearch-extension  --dev`

2. Remove any elasticsearch and kibana entries from your .lando.yml.

3. Install **Elasticsearch** extension

`lando load-wunderio-lando-drupal-extensions elasticsearch`

4. Install **Kibana** extension

`lando load-wunderio-lando-drupal-extensions kibana`

5. Rebuild lando

`lando rebuild`

6. To remove extensions (this was added in https://github.com/wunderio/lando-drupal/pull/28 )

`lando unload-wunderio-lando-drupal-extensions elasticsearch`
`lando unload-wunderio-lando-drupal-extensions kibana`

or

`lando unload-wunderio-lando-drupal-extensions elasticsearch,kibana`

### CODE REVIEW

File: extensions/elasticsearch/.lando.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+services:
+  elasticsearch:
+    type: compose
+    services:
+      image: "docker.elastic.co/elasticsearch/elasticsearch:7.17.0"
```
Comment (deiviic) [inline extensions/elasticsearch/.lando.yml]: Maybe we should set version to 8.11 already?

File: extensions/kibana/.lando.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+services:
+  kibana:
+    type: compose
+    services:
+      image: "docker.elastic.co/kibana/kibana:7.17.0"
```
Comment (deiviic) [inline extensions/kibana/.lando.yml]: Maybe we should set version to 8.11 already?

**Comment by deiviic** [comment]
Working as described, just curious about the version, if we should maybe use v8.11 already here.

File: extensions/elasticsearch/.lando.yml (hunk 1 above)
Comment (hkirsman) [reply extensions/elasticsearch/.lando.yml]: Yes, I agree. We should be keeping up to date with EOL dates. I moved these to _7 extensions: https://github.com/wunderio/lando-drupal/pull/29/commits/60575e2d4aae893656dd681a5fcbe0b899c701e4

image

The one without version is default.. or rolling version. If 8 would get EOL, we would update it. But I wonder if anybody dares to use it and we should have versions for all so elasticsearch would become elasticsearch_8

@deiviic

Comment (hkirsman) [reply extensions/elasticsearch/.lando.yml]: It would also maybe be more intuitive for somebody who is looking at this for the first time.

**Review by deiviic — APPROVED** [review]

File: extensions/elasticsearch/.lando.yml (hunk 1 above)
Comment (hkirsman) [reply extensions/elasticsearch/.lando.yml]: @deiviic I need to merge this branch and create a release but seems 8.x didn't start to work. I've created new issue dealing with that plus naming question https://github.com/wunderio/lando-drupal/issues/32

**Pull request merged**